CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 
 ### Fixed
+* Superfluous space between negate and RationalConst. [#2265](https://github.com/fsprojects/fantomas/issues/2265)
```

Comment:
I tend to add the new entry at the end of the list.
This really doesn't matter that much, just letting you know.